src/features/device-heartbeat/index.ts

```diff
@@ -43,16 +48,16 @@ const getPollIntervalForDevice = _.once(() =>
 				name: 'desc',
 			},
 		},
-	}),
-);
-
-const getPollIntervalForParentApplication = _.once(() =>
-	api.resin.prepare<
-		{
-			deviceId: number;
-		},
-		'application_config_variable'
-	>({
+	} as const satisfies Params<DeviceConfigVariable>;
```

Comment:
The `as const` in this pattern is necessary to make sure we get typings like `resource: 'device_config_variable'` instead of `resource: string` whilst the `satisfies Params<$resource>` means that it must be a valid `Params` object for that resource (since when passed to the `prepare` generic it doesn't actually enforce that for some reason)